hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -35,18 +36,48 @@ interface IMissingContainerResponse {
   pipelineID: string;
 }
 
+interface IAllContainerResponse {
```

Comment:
Some naming suggestions, maybe it is more clear what are these used for.
```suggestion
interface IContainerResponse {
```

Reply:
Done